src/main/resources/assets/cctweaks/lua/prebios-string.lua

```diff
@@ -71,4 +87,12 @@ if native_getfenv then
 	end
 end
 
-return fun()
+-- Try to run the bios, print possible errors
+local _, ok, err = pcall(fun)
```

Comment:
Oh yeah I misunderstood the pcall() function. Yeah the double check is not needed, sorry.